packages/components/src/components/primitives/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import {
+  Menu,
+  MenuButtonProps,
+  MenuProps,
+  MenuButton,
+  MenuListProps,
+  MenuList,
+  MenuItem,
+  MenuItemProps,
+  forwardRef,
+} from '@chakra-ui/react';
+import { DropdownStaticMembers } from './types';
+
+export const Dropdown: FC<MenuProps> & DropdownStaticMembers = (props) => {
```

Comment:
can we export this as a `forwardRef`?

Reply:
The `Menu` component does not have a `ref` prop.